### PR TITLE
Mobile nav tweak

### LINKED ISSUE #46: Mobile Navigation
They like the mobile layout, let's move forward with restructuring/doing whatever we need to do to get it aligning perfectly. How it looks here is perfect,
image

### PR DESCRIPTION
**Pull request by ogdenstudios**
Hey @baudoinalkali - I don't know what was in my head last week or the week before when I said we'd have to restructure the nav. Maybe I was just not seeing it. I'm pretty sure this one tweak fixes everything on mobile through tablet breakpoints as described in your Loom video. 

Will you take a look and let me know if I've missed something? Seems too easy, but it's looking pretty good!

Resolves #46

### CODE REVIEW

**Comment by ogdenstudios** [comment]
@baudoinalkali - my vacation brain got the best of me. This PR also includes the work for #49. I ended up committing on the same branch. They're related enough that I don't think it's worth it to go back and undo everything. 

Questions for you on the desktop navigation:

1. Should I make the green contact bar stick as well?
2. Do you want the navigation to be sticky on mobile? You specified desktop, so I kept it just desktop, but it would be easy to get a sticky nav for mobile as well with this configuration.

Resolves #49

**Comment by baudoinalkali** [comment]
@ogdenstudios Fixed desktop is good, can we add a drop shadow like they have on their current site when that happens? Mobile shouldn't be fixed, should have specified that.

**Comment by baudoinalkali** [comment]
@ogdenstudios Mobile is looking good, I noticed on my device it's showing as slightly uneven spacing. image

**Comment by ogdenstudios** [comment]
Got it. I'll add the dropshadow to the sticky nav and then see if I can tweak up that space a little.

File: components/header.js [hunk 1]
```diff
@@ -2,10 +2,27 @@ import DesktopNav from '../components/Navigation/DesktopNav'
 import MobileNav from '../components/Navigation/MobileNav'
 import { faMobileAlt } from "@fortawesome/free-solid-svg-icons"; // import the icons you need
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"; // Import the FontAwesomeIcon component
+import { useEffect, useRef } from 'react'
 
 const Header = () => {
+    const stickyNav = useRef(null);
+
+    const onScroll = () => {
+        const shadowClass = 'lg:shadow'
```
Comment (ogdenstudios) [inline components/header.js:11]: @baudoinalkali - I just used the [base level shadow utility](https://tailwindcss.com/docs/box-shadow), but if you want more or less, here is where you would tweak it. Just replace `lg:shadow` with `lg:shadow-md` or whatever else you prefer. 

If you know which one you want now, I can just add it as part of this PR.

Comment (baudoinalkali) [reply components/header.js:11]: @ogdenstudios What you have is fine.

**Comment by ogdenstudios** [comment]
@baudoinalkali - ok, I tweaked the mobile dropdown and I think it's dead even now as required. This is ready for your review and then I think we can merge it in and close out those issues.

**Pull request merged**